tests/sentry/integrations/slack/webhooks/actions/test_status.py

```diff
@@ -271,30 +271,6 @@ def test_ask_linking(self):
         assert resp.data["response_type"] == "ephemeral"
         assert resp.data["text"] == LINK_IDENTITY_MESSAGE.format(associate_url=associate_url)
 
-    def test_archive_issue(self):
```

Comment:
this was testing for backwards compatibility which is no longer necessary https://github.com/getsentry/sentry/pull/73248/files#diff-09dc9b9a6030a931823ed51799486b635133b203c044824cafb66debb62fe1c6L290